mcodingbot/utils/__init__.py

```diff
@@ -1,5 +1,5 @@
-import typing
+from typing import Sequence
```

Comment:
I dont think there should be a from import here

Reply:
Why?

Reply:
I thought this project did `import X`